lib/require-plugin.js

```diff
@@ -5,10 +5,10 @@ const utils = require('./utils');
 
 const isPluginNotFoundError = (e, pluginName) => {
     return [
-        `Error: Cannot find module '${pluginName}'`,
+        new RegExp(`Error: Cannot find module '${pluginName}'`),
```

Comment:
I don't like injecting unsanitized user input into regex (it may potentially throw exception if plugin name contains some special characters). I suggest just having two `if`s. `if (errorMessage.includes('Error: Cannot ...'))` and `if (/plugins-loader tried to .../.test(errorMessage))`